tests.py

```diff
@@ -27,9 +30,14 @@ def run_vsm(self, name, input_data, expected_output, use_initial=True):
 
         process = Popen(cmd, stdin=PIPE, stdout=PIPE)
 
-        output, _ = process.communicate(input=input_data.encode(), timeout=2)
-
-        self.assertEqual(output.decode() , expected_output)
+        output, _ = process.communicate(input=input_data.encode(), timeout=5)
```

Comment:
These changes are also not needed. What you need to do is change each test so that their expected_output matches what will now be printed.

Look at this:

```
     def test_simple0(self):
          input_data = 'transmission_gear = "reverse"'
         expected_output = 'car.backup = True\n'
         self.run_vsm('simple0', input_data, expected_output)
```

Do you see expected_output there? It has the output that is expected from VSM when this test runs. You need to edit that string to include what's now expected.